QuickFixers/Views/Client/ScheduledServices.cshtml

```diff
@@ -0,0 +1,64 @@
+﻿@model IEnumerable<QuickFixers.Data.Models.ScheduledService>
+
+@{
+    ViewBag.Title = "ScheduledServices";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<h2>ScheduledServices</h2>
+
+<p>
+    @Html.ActionLink("Create New", "Create")
+</p>
+<table class="table">
+    <tr>
+        <th>
+            @Html.DisplayNameFor(model => model.scheduledServiceID)
+        </th>
+        <th>
+            @Html.DisplayNameFor(model => model.serviceProviderID)
+        </th>
+        <th>
+            @Html.DisplayNameFor(model => model.clientID)
+        </th>
+        <th>
+            @Html.DisplayNameFor(model => model.servicesOfferedID)
+        </th>
+        <th>
+            @Html.DisplayNameFor(model => model.serviceDate)
+        </th>
+        <th>
+            @Html.DisplayNameFor(model => model.serviceAddress)
+        </th>
+        <th></th>
+    </tr>
+
+@foreach (var item in Model) {
+    <tr>
+        <td>
+            @Html.DisplayFor(modelItem => item.scheduledServiceID)
+        </td>
+        <td>
+            @Html.DisplayFor(modelItem => item.serviceProviderID)
+        </td>
+        <td>
+            @Html.DisplayFor(modelItem => item.clientID)
+        </td>
+        <td>
+            @Html.DisplayFor(modelItem => item.servicesOfferedID)
+        </td>
+        <td>
+            @Html.DisplayFor(modelItem => item.serviceDate)
+        </td>
+        <td>
+            @Html.DisplayFor(modelItem => item.serviceAddress)
+        </td>
+        <td>
+            @Html.ActionLink("Edit", "Edit", new { id = item.scheduledServiceID/* id=item.PrimaryKey */ }) |
+            @Html.ActionLink("Details", "Details", new { id = item.scheduledServiceID /* id=item.PrimaryKey */ }) |
+            @Html.ActionLink("Delete", "Delete", new { id = item.scheduledServiceID /* id=item.PrimaryKey */ })
```

Comment:
Please update links to go to an existing page like /Test or /Shared/Error(or maybe remove it we don't need any)

Reply:
Removed unnecessary links.